scripts/verify_hutch

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# function that takes in a string and verifies its a known hutch
+
+usage(){
+cat << EOF
+usage: ${BASH_SOURCE[0]} <hutch> [option]
+
+Script that returns confirmation if the passed hutch is a known/existing hutch, success indicated by exit 0, failure by exit 1. 
+
+EOF
+}
+
+foundHutch=1
+hutch=${1,,}
+if [ $# -lt 1 ]; then
+   echo 'need arguments: input hutch name' >&2
+   usage
+   exit 1
+fi 
+
+if [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+fi
```

Comment:
Just changing spacing and order to organize a bit better, argument checking should go before the variable assignments.
```suggestion
if [ $# -lt 1 ]; then
   echo 'need arguments: input hutch name' >&2
   usage
   exit 1
fi 
if [[ ($1 == "--help") || ($1 == "-h") ]]; then
   usage
   exit 0
fi

foundHutch=1
hutch=${1,,}

```